single-line-diagram/single-line-diagram-core/src/main/resources/ConvergenceLibrary/unknown-component.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" height="16" width="16">
+    <path d="M 9.9248465,0 0,10.015385 9.9556158,20.030769 19.999391,9.9538462 Z m 2.5705015,7.9302308 c 0,1.7033846 -0.851881,1.9296154 -2.196256,2.8611542 v 1.211 H 9.2741726 v -1.67677 C 10.432199,9.7267693 11.49705,9.261 11.470429,7.9967693 11.443808,6.3466154 8.8866164,6.577 7.6221055,7.3621539 L 7.9297978,7.3313847 V 6.1869231 C 9.9663259,5.5481539 12.495348,5.588077 12.495348,7.9302308 Z M 9.2434034,12.671154 h 1.1047836 v 1.224308 H 9.2434034 Z"/>
```

Comment:
Precision is too high for what is needed, I simplified it with a precision of 3:
```suggestion
    <path d="M 10,0 -0,10 10,20 20,10 Z m 2.57,7.93 c 0,1.7 -0.8,1.93 -2.2,2.87 V 12 H 9.28 V 10.3 C 10.5,9.73 11.5,9.26 11.5,8 11.5,6.35 8.89,6.58 7.62,7.36 H 7.93 V 6.19 C 9.97,5.55 12.5,5.59 12.5,7.93 Z M 9.25,12.7 h 1.15 v 1.2 H 9.25 Z"/>
```